collector/lib/CollectorConfig.cpp

```diff
@@ -160,6 +163,48 @@ CollectorConfig::CollectorConfig(CollectorArgs* args) {
   HandleAfterglowEnvVars();
 
   host_config_ = ProcessHostHeuristics(*this);
+
+  const char* envvar;
+
+  connection_stats_quantiles_ = {0.50, 0.90, 0.95};
+
+  if ((envvar = std::getenv("ROX_COLLECTOR_CONNECTION_STATS_QUANTILES")) != NULL) {
+    connection_stats_quantiles_.clear();
+    std::stringstream quantiles(envvar);
+    while (quantiles.good()) {
+      std::string quantile;
+      std::getline(quantiles, quantile, ',');
```

Comment:
Wait, `ROX_COLLECTOR_CONNECTION_STATS_QUANTILES` is a new line separated environment variable? That doesn't sound very ergonomic, why not use commas, colons, semicolons or some other printable character?

Reply:
This is a coma separated string. `getline()` is unfortunately badly named.